lms/resources/_js_config/__init__.py

```diff
@@ -35,9 +35,6 @@ def _h_user(self):
         return self._lti_user.h_user
 
     @property
-    def _consumer_key(self):
```

Comment:
Removed `_consumer_key`. 

Note that I carry `property` to `_application_instance` which I think it makes sense.